### DISCUSSION TITLE
Welcome to ASUS-FX504GE-Hackintosh_OpenCore_0.6.4 Discussions!

### DISCUSSION THREAD

**Discussion by RobyRew** [RobyRew/ASUS-FX504GE-Hackintosh_OpenCore#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nguyenndm1901** [comment]
Hello, was anyone here able to fix iServices? whenever I login to iMessage or FaceTime, it will return to login screen. Please help

**Reply by RobyRew** [reply]
You Need to download [Python 3](https://www.python.org/ftp/python/3.9.1/python-3.9.1-macosx10.9.pkg) and then download [GenSMBIOS](https://github.com/corpnewt/GenSMBIOS/archive/master.zip), and generate you'r own SMBIOS, preferable with _MacBookPro15,3_

Once you install python, you need to open _GenSMBIOS.zip_, and open de _GenSMBIOS.command_ located in the GenSMBIOS-master folder.

For more details check this video: https://youtu.be/8MsueH5EouQ?t=276

BTW, if you like my work, please give it a star 😉   ⭐️

**Reply by nguyenndm1901** [reply]
Thanks for your very fast answer. I did all the steps above already but iServices is still unable to use.
Here is my recording: https://youtu.be/XOyH_MGo1h0

**Reply by RobyRew** [reply]
> Thanks for your very fast answer. I did all the steps above already but iServices is still unable to use.
> Here is my recording: https://youtu.be/XOyH_MGo1h0

But, after you put the new SMBIOS you did a reboot?

Also take an eye here on [Dortania's Troubleshooting section](https://dortania.github.io/OpenCore-Post-Install/universal/iservices.html#generate-a-new-serial)



Edit: Now I remember that in some cases the system can select the WiFi as en0 because itlw.kexts simulate the wifi to be an ethernet, and this can cause problems if the real ethernet its not a real ethernet (Info [Here](https://dortania.github.io/OpenCore-Post-Install/universal/iservices.html#fixing-en0)), you need to do these 2 steps:

```
sudo rm /Library/Preferences/SystemConfiguration/NetworkInterfaces.plist
sudo rm /Library/Preferences/SystemConfiguration/preferences.plist
```

Be sure to install [Heliport app](https://github.com/OpenIntelWireless/HeliPort/releases/download/v1.3.1/HeliPort.dmg) for WiFi

**Reply by RobyRew** [reply]
> Thanks for your very fast answer. I did all the steps above already but iServices is still unable to use.
> 
> Here is my recording: https://youtu.be/XOyH_MGo1h0

Hey, you fixed, or it still have this problem?

**Reply by nguyenndm1901** [reply]
Sorry for not replying you. The problem is still there. I replaced itlw.kext with airportitlwm.kext v1.2.0 for bigsur and I did a clean snapshot. I did fix ethernet to en0 based on your steps. Please let me know if you want to control my laptop to try to fix it.

**Reply by RobyRew** [reply]
> Sorry for not replying you. The problem is still there. I replaced itlw.kext with airportitlwm.kext v1.2.0 for bigsur and I did a clean snapshot. I did fix ethernet to en0 based on your steps. Please let me know if you want to control my laptop to try to fix it.

noooo, [itlwm.kext](https://github.com/OpenIntelWireless/itlwm/releases/download/v1.2.0/itlwm_v1.2.0_stable.kext.zip) its necessary, airportitlwm.kext its a hard installation (delete from your EFI), just use [itlwm.kext](https://github.com/OpenIntelWireless/itlwm/releases/download/v1.2.0/itlwm_v1.2.0_stable.kext.zip)

**Reply by nguyenndm1901** [reply]
Actually I tried both either itlwm or airportitlwm but none of them worked, then I decided to use airportitlwm because Dortania's guide recommends it. Can you add [my Facebook](https://www.facebook.com/hamvui9910/) for contacting purpose?

**Reply by RobyRew** [reply]
Twitter, Reddit?
Im RobyRew everywhere

**Reply by RobyRew** [reply]
> Actually I tried both either itlwm or airportitlwm but none of them worked, then I decided to use airportitlwm because Dortania's guide recommends it. Can you add [my Facebook](https://www.facebook.com/hamvui9910/) for contacting purpose?

Hey can you try the new release?:
https://github.com/RobyRew/ASUS-FX504GE-Hackintosh_Updated_OpenCore/releases/tag/4.0

**Comment by zerrium** [comment]
Hello, I recently installed OpenCore BigSur on my FX504GE by cloning this repo and copy the EFI folder

Everything works fine except HDMI which you have mentioned before
But I have issue for the 3.5mm combo jack where the audio output is weird and it doesn't detect my mic

Anyone has the same issue?

**Reply by RobyRew** [reply]
Hello, I am glad to be useful

For HDMI im still trying configurations, I hope this year to found something better than the actual
I made a quick update on the EFI under de DeviceProperties hoping solves something.

For 3.5mm Jack I will search a pair of headphones and do some testings, when I have results I will tell you here or doing a Release

For now you can test this EFI:
[AsusFX504-RELEASE_FixHDMI.zip](https://github.com/RobyRew/ASUS-FX504GE-Hackintosh_Updated_OpenCore/files/6134440/AsusFX504-RELEASE.zip)

**Reply by zerrium** [reply]
Thanks for your quick response

If you dont mind, what changes did you make to that zip file? Is it same as the master branch of this repo?

**Reply by RobyRew** [reply]
Increased VRAM and synchronize with readme

**Reply by RobyRew** [reply]
> Thanks for your quick response
> 
> If you dont mind, what changes did you make to that zip file? Is it same as the master branch of this repo?

Hey can you try the new release and say if works better or the same please
https://github.com/RobyRew/ASUS-FX504GE-Hackintosh_Updated_OpenCore/releases/tag/4.0

**Reply by RobyRew** [reply]
> Hello, I recently installed OpenCore BigSur on my FX504GE by cloning this repo and copy the EFI folder
> 
> Everything works fine except HDMI which you have mentioned before But I have issue for the 3.5mm combo jack where the audio output is weird and it doesn't detect my mic
> 
> Anyone has the same issue?

Can you try now with this actual OpenCore 0.7.4-1 Build EFI?
for audio jack3.5 i use layout 15 because have HQ over jack3.5 and HDMI works perfectly with layout 15 the downside is that you lose speakers sound

**Comment by zerrium** [comment]
> Hello, I am glad to be useful
> 
> For HDMI im still trying configurations, I hope this year to found something better than the actual
> I made a quick update on the EFI under de DeviceProperties hoping solves something.
> 
> For 3.5mm Jack I will search a pair of headphones and do some testings, when I have results I will tell you here or doing a Release
> 
> For now you can test this EFI:
> [AsusFX504-RELEASE_FixHDMI.zip](https://github.com/RobyRew/ASUS-FX504GE-Hackintosh_Updated_OpenCore/files/6134440/AsusFX504-RELEASE.zip)


Update:

HDMI issue is still there, nothing changed


For 3.5mm audio jack I try more experiments to find out why the audio output is weird:

1. When I plugged my headphones, actually they sound weird because of the frequency response. Only some frequencies like high, or mid high coming from the headphones. Thus making the audio quality drops and sounds weird.

2. So about that issue, I try using an external active speaker and it sound normal, nothing weird and fine. I can hear all frequencies coming from the speaker.

3. I try using my TWS (connection with bluetooth) and my TWS sound fine, no issue at all, the audio quality is fine like it supposed to.

4. From those experiments my conclusion is: the 3.5mm audio jack can't drive a high impedance headphones. FYI mine is a gaming headphones with 39 Ohm impedance. On windows 10, it can drive them just fine so perhaps this is just the kext issue (?)


Oh yeah also it still can't detect external mic plugged to the 3.5mm jack and also can't detect incoming audio from my TWS mic

Hopefully this helps you fix the problems. Mac OS is not my main OS on my laptop, just for development purpose because I'm learning iOS apps development using Xcode which is not available on other platforms beside Mac OS.

Finally, I want to thank you so much from making this OS stable, previously I tried using High Sierra Clover and ohh man it crashed a lot and sometimes it just kernel panic.

So far, trying this BigSur OpenCore from your patch seems stable, no crash, no lag, no freezing at all and no kernel panics.

About those issues I mentioned earlier, just take your time, I don't rush you to fix them I just help you to analyze why it occurs.

That's all from me and have a good day

**Comment by deg-jc** [comment]
Hi @RobyRew , Thanks a lot for doing this great great stuff for us. I wish to contact you Roby but I Can't find how, I've managed to install BigSur on this same Laptop but I can't play videos either volume, backlight and that kind of keys doesn't work & the laptop takes too long to start after like 3 minutes black screen. Please, I wonder if we can have a talk. I'm working with OC 0.6.9.
**My email: diego.jerezcast@gmail.com**

**Reply by RobyRew** [reply]
Yeah i know they are a few issues, im planing to update to OPenCore 0,6.9 these days and fixim backlight problem
Also updating the guide for more easy understanding.
Thanks for your feedback!

**Reply by deg-jc** [reply]
I would love to help since this is my laptop for my work, I got hacked and had to delete Mojave done with Clover but without wifi, bluethoot, hdmi or touchpad, worked for almost 2 years very well.Someone did it for me.But I want to have it as clear as possible, so please let me help on this guide Roby, I really appreciate if you let me, just asking some patience as I'm new to this but I can learn.

**Reply by RobyRew** [reply]
> I would love to help since this is my laptop for my work, I got hacked and had to delete Mojave done with Clover but without wifi, bluethoot, hdmi or touchpad, worked for almost 2 years very well.Someone did it for me.But I want to have it as clear as possible, so please let me help on this guide Roby, I really appreciate if you let me, just asking some patience as I'm new to this but I can learn.

Hello @deg-jc I have fixed backlight on this newer release, the boot time should be 20 seconds, you can test this new version released a few minutes before:
https://github.com/RobyRew/ASUS-FX504GE-Hackintosh_Updated_OpenCore/releases/tag/4.0
for brightness kesy I will make a great post install tutorial in the next future, its easy to change the keys on system preferences
